homeassistant/components/fritz/common.py

```diff
@@ -372,7 +373,7 @@ def scan_devices(self, now: datetime | None = None) -> None:
 
         mesh_intf = {}
         # first get all meshed devices
-        for node in topology["nodes"]:
+        for node in topology.get("nodes", ""):
```

Comment:
Would suggest to default to an empy list, but not an empty string 🤔 
```suggestion
        for node in topology.get("nodes", []):
```